services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/permissions/HankeKayttajaService.kt

```diff
@@ -95,7 +95,7 @@ class HankeKayttajaService(
 
         val hankeId = hanke.id ?: throw HankeArgumentException("Hanke without id")
         val hankeTunnus = hanke.hankeTunnus ?: throw HankeArgumentException("Hanke without tunnus")
-        val hankeNimi = hanke.nimi ?: throw HankeArgumentException("Hanke without name")
+        val hankeNimi = hanke.nimi
```

Comment:
This line can possibly be removed? hankeNimi was there just to make sure nimi is not null.

Reply:
Thank you for noticing. I inlined the variable.